caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -262,7 +262,7 @@ def rpc_termination_callback():
                     )
                     if isinstance(model_future, LocalModelTrainer.LocalModelFuture):
                         raise model_future._worker.error
-                    raise RuntimeError(training_info.errors[0])
+                    raise training_info.errors[0]
```

Comment:
can you fix the `_worker` stuff above?

Reply:
(it should no longer be needed)

Reply:
We are still not raising the error in `wait()`. so it will still be needed?  Or you mean the if condition at line 264?